schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+    id INTEGER,
```

Comment:
[ OPTIONAL CHANGES ]

- [ ] It would be nice if you could set the ID NOT Null and to auto-increment like this `id int NOT NULL IDENTITY(1,1)`
- [ ] Also to set the ID a primary key like this `PRIMARY KEY (id)`